Annotato/Annotato/Views/Document/Edit/Annotation/AnnotationPaletteView.swift

```diff
@@ -103,12 +104,15 @@ extension AnnotationPaletteView {
 
     @objc
     private func didTapEditOrViewButton(_ button: UIButton) {
-        if editOrViewButton.isSelected {
-            editOrViewButton.isSelected = false
-            viewModel.enterViewMode()
-        } else {
-            editOrViewButton.isSelected = true
+        editOrViewButton.isSelected.toggle()
+
+        let isNowEditing = editOrViewButton.isSelected
+        if isNowEditing {
             viewModel.enterEditMode()
+            viewModel.enterMaximizedMode()
+        } else {
+            viewModel.enterViewMode()
+            viewModel.enterMinimizedMode()
```

Comment:
On entering view mode, is it better to leave it maximised (if it was initially maximized), since they may want to view their entire annotation?

Reply:
Okay will do that